### ISSUE TITLE
Use CUDA in Shuffling 

### ISSUE THREAD

**Issue by yazeedalrubyli** [yazeedalrubyli/Xtract#3]
Use CUDA in Shuffling the data, because it takes so long with regular sklearn.utils.shuffle

**Comment by yazeedalrubyli** [comment]
Sklearn Shuffle Replaced by Keras Shuffle

**Comment by yazeedalrubyli** [comment]
Sklearn Shuffle Replaced by Keras Shuffle
